### DISCUSSION TITLE
Morph relations saves *able_type column as model class' names.

### DISCUSSION THREAD

**Discussion by tolgabrandt** [laravel/framework#45851] [Q&A] [answered]
When i create a morph relational record with Eloquent i see directly the model class names in type columns. When i implement another model methods with "wherepivot", i can not get records because some backslashes not match with class names. How can i change morph relations to table name globally?

**Comment by denizgolbas** [comment, marked as answer]
In your AppServiceProvider boot method(or you can create another provider):
Add:
```
use Illuminate\Database\Eloquent\Relations\Relation;
```
```php
Relation::morphMap([
    'posts' => 'App\Models\Post',
    'videos' => 'App\Models\Video',
]);
```
More details: https://laravel.com/docs/master/eloquent-relationships#custom-polymorphic-types
